src/Voron/Impl/Transaction.cs

```diff
@@ -714,8 +714,8 @@ public void ForgetAbout(in long storageId)
 
             if (_cachedDecompressedBuffersByStorageId.Remove(storageId, out var t))
             {
-                Allocator.Release(ref t);
                 _lowLevelTransaction.DecompressedBufferBytes -= t.Length;
+                Allocator.Release(ref t);
```

Comment:
Can you please explain this change? What was the issue?

Reply:
t.Length is 0 after `Allocator.Release(ref t);` so we decrement `_lowLevelTransaction.DecompressedBufferBytes` by 0 instead of the actual `t.Length`